components/builder-api/src/config.rs

```diff
@@ -210,6 +212,28 @@ impl ToSocketAddrs for HttpCfg {
     }
 }
 
+#[derive(Clone, Debug, Deserialize)]
+#[serde(default)]
+pub struct KafkaCfg {
+    pub bootstrap_nodes:     String, // "foo.bar.baz:9090,baz.fuzz.bizz:9191"
+    pub client_id:           String,
+    pub connection_retry_ms: u64,
+    pub message_timeout:     String,
```

Comment:
For timeouts and retry limits, I've found `Duration` to be a more useful type, particularly because it encapsulates the time units within itself, so you don't have to have names like "foo_ms" and "bar_min".